concent_api/requirements.lock

```diff
@@ -18,6 +18,7 @@ python-dateutil==2.6.1
 pytz==2017.3
 PyYAML==3.12
 repoze.lru==0.7
+requests==2.18.4
```

Comment:
Don't edit `requirements.lock` manually. Recreate it with `pip freeze` in a fresh virtualenv. You're missing the dependencies of `requests` here (`certifi`, `urllib3`, etc.)